src/assets/ts/keyEmitter.ts

```diff
@@ -107,7 +107,16 @@ export default class KeyEmitter extends EventEmitter {
   }
 
   public listen() {
+    // IME event
+    $(document).on('compositionend', (e: any) => {
+      this.emit('keydown', e.originalEvent.data);
```

Comment:
Would it work to do

```
e.originalEvent.data.split('').forEach((key) => {
  this.emit('keydown', key);
});
```

here, instead of handling the IME case in registerMode?

Reply:
Okay. That's better.